src/algorithms/time_evolution/simpleupdate.jl

```diff
@@ -28,24 +28,24 @@ Simple update of the x-bond between `[r,c]` and `[r,c+1]`.
 ```
 """
 function _su_xbond!(
-        peps::InfinitePEPS,
-        gate::AbstractTensorMap{T, S, 2, 2},
-        env::SUWeight,
-        row::Int,
-        col::Int,
-        trscheme::TruncationScheme,
-    ) where {T <: Number, S <: ElementarySpace}
-    Nr, Nc = size(peps)
+        state::P, gate::AbstractTensorMap{T, S, 2, 2}, env::SUWeight,
+        row::Int, col::Int, trscheme::TruncationScheme; gate_ax::Int = 1
+    ) where {P <: InfiniteState, T <: Number, S <: ElementarySpace}
```

Comment:
```suggestion
        state::InfiniteState, gate::AbstractTensorMap{T, S, 2, 2}, env::SUWeight,
        row::Int, col::Int, trscheme::TruncationScheme; gate_ax::Int = 1
    ) where {T <: Number, S <: ElementarySpace}
```
This doesn't really need a where clause, and I think this reads slightly better